package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "staker",
+  "version": "1.0.0",
+  "description": "Staker is a flexible, configurable staking contract. Staker makes it easy to distribute onchain staking rewards for any ERC20 token, including DAO governance tokens. This file is included for compatibility with npm based development frameworks such as Hardhat.",
+  "author": "Contributors to https://github.com/ScopeLift/staker",
+  "files": [
+    "src/*"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ScopeLift/staker.git"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ScopeLift/staker/issues"
+  },
+  "homepage": "https://github.com/ScopeLift/staker#readme",
+  "dependencies": {
+    "forge-std": "latest",
+    "openzeppelin-contracts": "latest"
```

Comment:
This package looks outdated. I searched npm and this seems to be the more up to date version: https://www.npmjs.com/package/@openzeppelin/contracts.

We should also try to pin to the same version that we're using in forge.

Reply:
Fix: https://github.com/withtally/staker/pull/121/commits/c9401fa462b9f79aa7c5226ddb3f74978db7af49